src/rotation.zig

```diff
@@ -18,6 +18,13 @@ const RayIntersectionResult = struct {
 	max: Vec3f,
 };
 
+pub const Degrees = enum(u2) {
+	@"90" = 0,
+	@"180" = 1,
+	@"270" = 2,
+	@"0" = 3,
```

Comment:
shouldn't 0° be mapped to 0?
That's also how you are using it, if I interpret this correctly:
https://github.com/PixelGuys/Cubyz/blob/150ac3c71d6dc2e604cf8050a6d77e20610ed004/src/rotation.zig#L200-L203

Reply:
Yes, definitely, I was experimenting with having zero covered by condition, but I ended up just making tables for everything.